config/credentials.yml.enc

```diff
@@ -1 +1 @@
-AfDwuFZprPp9Z4I8DWTzEhxKI0BqPrSstY2mOtKOYy9SNMX4XNFJBIjPJJRyY9ZJprbt20XZrsase725tiURaRViWO0gugOKAD45YB8CoW3mNhdp6FP+IZyM4R7TWGp0FNkBR+C1oo2n0dSz2MVA4FbIeHJoS52OzgvCbAj2kcT4PzrFszMmsEpqPCeOXNlOdiPaIwaRVIraZuZlLBR/T4s54t4I2o0t1iJz3OlbFU+M0U/T1al23aPBIe7dEsE9YClvpoZ5e7VriJcj0hQszOSB+i7HWDzE6Tt6nJuaf+FOpIj2L72BeNDZXzxfFHKebUXTCI4u+3rk1MCTUz1NsoPCr5nwtqu+CNaMkUksTKNOTzPVrvApikU7Dtrt9stccDOv61pW028fAuN6SlbnjueukReEKdJRDd4X--85ubgNoq1TFbE3kg--mH+Os3N6aQE1tiveYeu0Sg==
\ No newline at end of file
+wMp5ZSeNBrFeNmQyK6pXQC/FQHcd9M4NQug7t4QokM54Ml13lXBrDIAIN5vXexGmeq7a25j7PhGjCzk0nqi6kWgG4GYnrxzNkLrAHxhXYslEOQTcjQIWJj4lPSvH9Klnz6q7YCivSLGdACbjOUdhdsTbgOkS5LDvSEoS/D1TIQpdkXTEIbLGEQ9X+XGcAK50e3cFqwELvpLGlkPUORUngivIqK5MNIjRmEivL11LJIjMlwr/ZBrAwKSNb6BFZpaBDUhZSSkfkzZnam0HXMwTP2l0RJqBF9rdmJz1qYWOy6q3ay8WGLz0q7LGeswgMxnxusATjIbu44ftCap0wUuwwyuPBQyx4CXwi9oXytV+aGw9jmsl6z8s7grgeWsfCFTKx8XXntfmkmRXxE/ccIGRYowv6OQkf9rkaQ/1hJ0+OCgfQQ8rzGN9RGfYGbAsjE7jZmOad/q0Cg4wnpwZ/PxRivGyNiOEOIeYTBF+eltNJSYNhIdYML1cyZlcmmhwf1qZgv7DcdZMOcioandh+HmQvCv3/9Ro5h4pgfsrFjzCY8WYPsH9t1fZ2jM7caTBwqDBhkucD5u8sJVu2FvN6jJCQNhfW11sTePv+jXYfQtab9UgQi5WL6GXDW30lHuLo+f7l9D/7fK+7fq0H95hU1M=--chlG/QFtJlRCEqUh--V3GPFiKrjV8j8JKV/HRIYA==
```

Comment:
This was added in from Rileys PR, which adds the necessary keys for encrypted attributes